python/pylibcudf/pylibcudf/tests/test_string_wrap.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+import textwrap
+
+import pyarrow as pa
+import pylibcudf as plc
+from utils import assert_column_eq
+
+
+def test_wrap():
+    pa_array = pa.array(
+        [
+            "the quick brown fox jumped over the lazy brown dog",
+            "hello, world",
+            None,
+        ]
+    )
+    result = plc.strings.wrap.wrap(plc.interop.from_arrow(pa_array), 12)
```

Comment:
Minor nit: saving 12 as a variable would be better than hardcoding it twice.

Reply:
Will include this change in https://github.com/rapidsai/cudf/pull/16971